themes/custom/droopler_theme/scss/components/_d_blog_listing.scss

```diff
@@ -130,20 +132,7 @@ $d-blog-article-row-min-height: 420px !default;
 }
 
 .page-wrapper .blog-listing-main-header {
-  background-color: $d-blog-listing-header-color;
-  color: $color-odysseus;
-  padding-top: $d-blog-listing-padding-top;
-  padding-bottom: $d-blog-listing-padding-bottom;
-  text-transform: none;
-  text-align: center;
-
-  &:after {
-    @include decoration-header-line-main-center;
-  }
-  h1 {
-    color: $color-odysseus;
-    margin-bottom: $d-blog-title-margin-bottom ;
-  }
+  @include listing-main-header($d-blog-listing-header-color, $color-odysseus, $d-blog-listing-padding-top, $d-blog-listing-padding-bottom, $d-blog-title-margin-bottom);
```

Comment:
Let's add another variable `$d-blog-listing-header-text-color: $color-odysseus !default;` and pass it to the mixin instead of the color itself (same for product)